docs/operating-system/debugging.md

```diff
@@ -17,7 +17,7 @@ SSH access through the [SSH add-on] (which will give you SSH access through port
 
 ### Home Assistant Operating System
 
-Use a USB drive formatted with FAT, ext4, or NTFS and name it CONFIG (case sensitive). Create an `authorized_keys` file (no extension) containing your public key, and place it in the root of the USB drive. File needs to be ANSI encoded (not UTF-8) and must have Unix line ends (LF), not Windows (CR LF). See [Generating SSH Keys](#generating-ssh-keys) section below if you need help generating keys. Use the CLI (eg. SSH to the [SSH add-on] on port 22) and import the `authorized_keys` file with the `ha os import` command. You can now access your device as root over SSH on port 22222. Alternatively, the file will be imported from the USB when the Home Assistant OS device is rebooted.
+Use a USB drive formatted with FAT, ext4, or exFAT (NFTS wont work) and name it CONFIG (case sensitive). Create an `authorized_keys` file (no extension) containing your public key, and place it in the root of the USB drive. File needs to be ANSI encoded (not UTF-8) and must have Unix line ends (LF), not Windows (CR LF). See [Generating SSH Keys](#generating-ssh-keys) section below if you need help generating keys. Use the CLI (eg. SSH to the [SSH add-on] on port 22) and import the `authorized_keys` file with the `ha os import` command. You can now access your device as root over SSH on port 22222. Alternatively, the file will be imported from the USB when the Home Assistant OS device is rebooted.
```

Comment:
Why doesn't it work? Is this a known issue, a known limitation, or a bug?

Reply:
The import-command does say the config files where imported successfully. Due to a lack of debug output I can only guess successfully meant "no files found, no errors, finished successfully"....
With the online help in mind I would have called it a bug (maybe NTFS-Drivers not being installed, on the other hand NTFS-Drivers are probably not needed on HA...)

I struggled a while and found a blogpost somewhere telling me to try FAT32 because NFTS does not work - from that point of view at least some people know of the limitation...

Reply:
Right, the point with my comment is: Bugs should be raised and triaged and not fixed by just adjusting the docs. Did you report this?

Reply:
No I did not, but sounds like the obvious thing to do instead of updating the docs.
And even worse: I was starting to take screenshots for the Bugtracker step by step of how to reproduce the issue.... I couldn't reproduce the issue...
Even though reformatting as exFAT solved the issue for me two weeks ago, reformatting as NTFS again importet my authorized_keys like a charm :-/

Thanks for double checking!

Fijne jaarwisseling!

Reply:
No problem! Happy New Year to you too 🥂